lib/yelp/responses/models/rating.rb

```diff
@@ -3,8 +3,35 @@
 module Yelp
   module Response
     module Model
-      class Rating < Response::Base
-        attr_reader :rating, :img_url
+      class Rating
+        attr_reader :rating
+
+        def initialize(json)
+          @rating = json['rating']
+          @image  = { regular: json['rating_image_url']       || json['rating_img_url'],
```

Comment:
This is goofy enough I'd probably include a comment, mentioning that different (endpoints? objects?) call this different things.